src/shogun/lib/any.h

```diff
@@ -111,23 +151,23 @@ namespace shogun
         {
             *(storage) = new T(*reinterpret_cast<T const*>(v));
         }
-        
+
```

Comment:
I agree with this change, but it should be not in this PR, as it makes it harder to review